src/Type.h

```diff
@@ -356,6 +356,16 @@ struct Type {
         return Type(code(), bits(), new_lanes, handle_type);
     }
 
+    /** Return Type with the same type code and number of lanes, but with twice as many bits. */
+    Type wide() const {
```

Comment:
Stylistically, `widen()` seems like a better name than `wide()` here IMHO.

Reply:
`widen` sounds like it works in-place? When reading the code, I would expect `t.widen()` to modify `t`, not return a wider `t`.

Reply:
The other thing I considered was `wider` but thought `wide` was better.

Reply:
> I would expect t.widen() to modify t, not return a wider t

What can I say, my expectations are the opposite. To me `wide()` reads as a noun, not a verb, which is weird for a method. I can live with `wide()` but might be worth a straw poll on Gitter to see if there is a strong consensus either way.

(Complete threadjack but `Type` is a class that I'd consider just making immutable to avoid this sort of confusion; it's small enough that just making mutated copies seems sensible.)

Reply:
I renamed it, I think either one works.